model/task/task_test.go

```diff
@@ -3669,6 +3669,156 @@ func TestArchive(t *testing.T) {
 	}
 }
 
+func TestArchiveFailedOnly(t *testing.T) {
+	defer func() {
+		assert.NoError(t, db.ClearCollections(Collection, OldCollection, event.LegacyEventLogCollection))
+	}()
+
+	resetDatabase := func() Task {
```

Comment:
would it be possible to split the test into multiple unit tests instead of calling reset database?

Reply:
The reset database call only happens once- I used it to help differentiate between what code is doing what. The tests are connected to each other and are to test the compounding of running archive over a couple iterations.

Reply:
i think its a little strange that a function called resetDatabases arbitrarily returns the display task so would it be possible to not return the display task in that case?